src/helpers/index.js

```diff
@@ -16,12 +16,14 @@ export function slideUnit(visibleSlides = 1) {
   return 100 / visibleSlides;
 }
 
-export function slideSize(totalSlides, visibleSlides) {
-  return ((100 / totalSlides) * visibleSlides) / visibleSlides;
+export function slideSize(totalSlides, visibleSlides, infinite) {
```

Comment:
Instead of passing "infinite" does it make more sense the adjust the "totalSlides" with the additional math?